src/material/checkbox/_checkbox-theme.scss

```diff
@@ -92,6 +92,16 @@
   }
 }
 
+@mixin customizations($tokens: ()) {
```

Comment:
Add documentation above similar to the other mixins (e.g. what this does, what the $tokens param is)